src/main/kotlin/wordle/view/OutputView.kt

```diff
@@ -0,0 +1,27 @@
+package wordle.view
+
+import wordle.domain.Tile
+
+object OutputView {
+
+    fun printStartMessage() {
+        println("WORDLE을 6번 만에 맞춰 보세요.")
+        println("시도의 결과는 타일의 색 변화로 나타납니다.")
```

Comment:
wordle의 라운드수가 바뀌게되면 view도 수정해야 되겠네요!
```suggestion
    fun printStartMessage(int round) {
        println("WORDLE을 $round번 만에 맞춰 보세요.")
        println("시도의 결과는 타일의 색 변화로 나타납니다.")
```

Reply:
좋은 의견 감사합니다 애쉬🙇‍♂️
애쉬 의견대로 수정해보겠습니다!